profiles/common/modules/features/multisite_crop_and_resize/multisite_crop_and_resize.module

```diff
@@ -313,6 +314,7 @@ function multisite_crop_and_resize_media_format_form_prepare_alter(&$form, $form
 
     // Create width, height, x, y fields.
     foreach (array('width', 'height', 'x', 'y', 'rotate') as $name) {
+      // @codingStandardsIgnoreStart : Ignore the variable translation
       $form[$name] = array(
         '#title' => t(ucfirst($name), array(), array('context' => 'Media crop')),
```

Comment:
Text book example of when it is allowed to ignore coding standards check, great job!